repl/repl.go

```diff
@@ -113,74 +113,68 @@ func (r *REPL) completer(d prompt.Document) []prompt.Suggest {
 }
 
 func (r *REPL) Input(t string) error {
-	_, err := r.executeLine(t)
-	return err
+	return r.executeLine(t)
 }
 
 // input processes a line of input and prints the result.
 func (r *REPL) input(t string) {
-	v, err := r.executeLine(t)
-	if err != nil {
+	if err := r.executeLine(t); err != nil {
 		fmt.Println("Error:", err)
-	} else if v != nil {
-		fmt.Println(v)
 	}
 }
 
 // executeLine processes a line of input.
 // If the input evaluates to a valid value, that value is returned.
-func (r *REPL) executeLine(t string) (values.Value, error) {
+func (r *REPL) executeLine(t string) error {
 	if t == "" {
-		return nil, nil
+		return nil
 	}
 
 	if t[0] == '@' {
 		q, err := LoadQuery(t)
 		if err != nil {
-			return nil, err
+			return err
 		}
 		t = q
 	}
 
 	astPkg := parser.ParseSource(t)
 	if ast.Check(astPkg) > 0 {
-		return nil, ast.GetError(astPkg)
+		return ast.GetError(astPkg)
 	}
 
 	semPkg, err := semantic.New(astPkg)
 	if err != nil {
-		return nil, err
-	}
-
-	r.scope.SetReturn(nil)
-
-	if _, err := r.interpreter.Eval(semPkg, r.scope, flux.StdLib()); err != nil {
-		return nil, err
+		return err
 	}
 
-	v := r.scope.Return()
-
-	// Ignore statements that do not return a value
-	if v == nil {
-		return nil, nil
+	ses, err := r.interpreter.Eval(semPkg, r.scope, flux.StdLib())
```

Comment:
I think you can condense this line and the lines above into a single call to `flux.Eval(t)`.